webapp/src/css/inbox.less

```diff
@@ -206,10 +225,12 @@ body {
   }
 }
 
-.filters {
-  height: @filters-bar-desktop-height;
+.tool-bar {
```

Comment:
Something broke the search in reports for mobile display
![Screenshot 2024-09-24 at 11 29 47 AM](https://github.com/user-attachments/assets/9d622b30-6024-4ca2-9eb1-0ca3ec45bc64)